devolutions-host/src/main.rs

```diff
@@ -0,0 +1,46 @@
+// Start the program without a console window.
```

Comment:
Nothing fancy here yet; Just logic to initialize log and config and copy-pasted test logic for DVC from MSRDPEX